apollo-server/apollo-server.ts

```
@@ -0,0 +1,45 @@
+    signUp: (_: unknown, { id}: User): string =>  `User registered successfully! ID: ${id}`
+  },
+};
+
+
+const server:ApolloServer = new ApolloServer({ typeDefs, resolvers });
+
+async function startServer():Promise<void> {
+  const { url } = await startStandaloneServer(server, { listen: { port: 4000 }}
+  );
+
+  console.log(`🚀 Server ready at ${url}`);
```

ESLint

[error] Unexpected console statement.

(no-console)